Formula/sorbet.rb

```diff
@@ -0,0 +1,27 @@
+class Sorbet < Formula
+  desc "Fast, powerful type checker designed for Ruby"
+  homepage "https://sorbet.org"
+  url "https://github.com/sorbet/sorbet/archive/refs/tags/0.5.10439.20220916051958-58983475a.tar.gz"
+  sha256 "ef751bd209c2bc9a35d91e1d4b391489ab8a51e9319e2113bcc0eace1cd0cf7b"
+  license "Apache-2.0"
+
+  depends_on "autoconf" => :build
+  depends_on "bazelisk" => :build
+  depends_on "parallel" => :build
+
+  def install
+    system "./bazel", "build", "//main:sorbet", "--config=release-#{OS.mac? ? "mac" : "linux"}"
```

Comment:
If I read https://github.com/sorbet/sorbet/blob/master/.bazelrc correctly this pulls in it's own LLVM at least, which is not acceptable for Homebrew

Reply:
I am not really familiar with Bazel. Are there workarounds?

Reply:
I don't know if there are

Reply:
Their `bazel` script, if I'm reading it correctly, also requires Rosetta: https://github.com/sorbet/sorbet/blob/58983475a4feb864551b8023b10f908bdb508b07/bazel#L59-L62

We don't want to assume Rosetta is installed on ARM machines. Additionally, our CI will soon not have it, so this will likely fail to build on ARM at some point.

Reply:
Asked upstream https://github.com/sorbet/sorbet/issues/6397 That's all I can do.

If not the dependency problem, I can confirm it works from my tap.
```
brew tap abitrolly/tap
brew install abitrolly/tap/sorbet
```

Reply:
bazel document the LLVM toolchain in here, https://github.com/sorbet/sorbet/blob/master/WORKSPACE#L17-L26